modules/apps/frontend-taglib/frontend-taglib/src/main/java/com/liferay/frontend/taglib/servlet/taglib/ComponentTag.java

```diff
@@ -197,8 +199,14 @@ protected boolean isPositionInline() {
 	private String _getRenderInvocation(String variableName) {
 		StringBundler sb = new StringBundler(14);
 
+		String componentId = Optional.ofNullable(
+			getComponentId()
+		).orElse(
+			_UNNAMED_COMPONENT_NAME + PortalUUIDUtil.generate()
```

Comment:
Hey @victorg1991, I think this is not an impeditive but react:component tag registers and apply the same pattern but in this case they [consider portletId and append an integer at the end of the identifier](https://github.com/liferay/liferay-portal/blob/46b2f042323f4053ddc76ba4afa46262047db915/modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/render.tsx#L61)

Reply:
Hey @diegonvs I believe this is to add more entropy to the id, since in JS we don't have any UUID generator. Do you think it is worth it to add it here? 🤔